README.md

```diff
@@ -1,56 +1,15 @@
-
-This example command line assumes the source files are located in the `/home/test` folder of the local machine.
-The name of the image to run is `agpipeline/canopycover:3.0`.
-
-We are using the same folder for the source metadata and the cleaned metadata.
-By using multiple `--mount` options, the source and output files can be separated.
-
-**Docker commands** \
-Everything between 'docker' and the name of the image are docker commands.
-
-- `run` indicates we want to run an image
-- `--rm` automatically delete the image instance after it's run
-- `--mount "src=/home/test,target=/mnt,type=bind"` mounts the `/home/test` folder to the `/mnt` folder of the running image
-- `-e "BETYDB_URL=https://terraref.ncsa.illinois.edu/bety/"` the URL to the BETYdb instance to fetch plot boundaries, and other data, from
-- `-e "BETYDB_KEY=<key value>"` the key associated with the BETYdb URL (replace `<key value>` with value of your key)
-
-We mount the `/home/test` folder to the running image to make available the file to the software in the image.
-
-**Image's commands** \
-The command line parameters after the image name are passed to the software inside the image.
-Note that the paths provided are relative to the running image (see the --mount option specified above).
-
-- `--working_space "/mnt"` specifies the folder to use as a workspace
-- `--metadata "/mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata.json"` is the name of the source metadata to be cleaned
-- `--citation_author "<author name>"` the name of the author to cite in the resulting CSV file(s)
-- `--citation_title "<title>"` the title of the citation to store in the resulting CSV file(s)
-- `--citation_year "<year>"` the year of the citation to store in the resulting CSV file(s)
-- `"/mnt/rgb_mask_L2_my-site_2018-10-01__14-20-40_mask.tif"` the names of one or more image files to use when calculating plot-level canopy cover
-
-# Original README
-Below are the contents of the original README.md for the TERRA REF project.
-It is included here as reference material for the history of this transformer.
-
-## Transformer Canopy Cover
-Calculated canopy cover (percentage of green pixels) on a per-plot basis with geometries retrieved from a BETYdb instance.
+Calculates canopy cover (the percentage pixels identified as a plant) on a plot level for one or more images
 
 ## Authors
 
 * Zongyang Li, Donald Danforth Plant Science Center, St. Louis, MO
 * Maxwell Burnette, National Supercomputing Applications, Urbana, Il
 * Robert Pless, George Washington University, Washington, DC
-
+* Chris Schnaufer, University of Arizona, Tucson, AZ
 
 ## Overview
+
 This extractor processes binary stereo images and generates values of plot-level percent canopy cover traits that are inserted into the BETYdb trait database.
 
  The core idea for this extractor is a plant-soil segmentation. 
```

Comment:
I think the Input and Output portions just below should be removed since the detailed description on how to use the Transformer are below; perhaps replace with a link to the `Use` section?

Reply:
Removed Input and Output sections